packages/hagrid/hagrid/cli.py

```diff
@@ -1890,3 +1896,19 @@ def generate_sec_random_password(length: int, alphabet: str = DEFAULT_ALPHABET)
 
     # Python 3 (urandom returns bytes)
     return "".join(alphabet[c % len(alphabet)] for c in urandom(length))
+
+
+# add Hagrid info to the cli
```

Comment:
I think users expect to type something like `version` and in that case perhaps we can just print the hagrid version since all the other information is already available under `hagrid debug`.

Reply:
@madhavajay I was actually thinking the same since we have `hagrid debug` happening, we could add anything there and keep `--version` to just print hagrid version. I'll do that.